lib/mox/application.ex

```diff
@@ -4,7 +4,10 @@ defmodule Mox.Application do
   use Application
 
   def start(_, _) do
-    children = [Mox.Server]
+    children = [
+      Mox.Server,
+      {Task.Supervisor, name: MoxTests.TaskSupervisor},
```

Comment:
**Annotation:** This supervisor is **only** used within unit tests so that I can call the `Task.Supervisor.async_nolink` method within those tests. If there's a better way to set this up that is 100% ok with me! I just went with the first working solution that I found.

Reply:
You can use `start_supervised!` in tests to start a task supervisor and then spawn under it.

Reply:
Ah! Great call. I switched to `start_supervised!` in 31217186dc7b618ea35448d2175377408a574fb3